pkg/sync/rpc/server.go

```diff
@@ -155,6 +157,7 @@ func NewSyncAgentServer(startPort, endPort int, replicaAddress, volumeName, inst
 	server := grpc.NewServer(interceptor.WithIdentityValidationReplicaServerInterceptor(volumeName, instanceName))
 	enginerpc.RegisterSyncAgentServiceServer(server, sas)
 	reflection.Register(server)
+	profilerrpc.RegisterProfilerServer(server, profiler.NewServer(volumeName+"-sync-agent"))
```

Comment:
What's the purpose of this?

Reply:
Can you put it in another PR or another commit? Also add this to https://longhorn.io/docs/1.9.1/troubleshoot/

Reply:
Sound good! Can you create another IMPROVEMENT ticket and PR for this improvement? Also add this to https://longhorn.io/docs/1.9.1/troubleshoot/. Thank you.

Reply:
Will remove this here and send another PR.